tests/test_index_option.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import sys
+
+import numpy as np
+
+sys.path.append("local")
+import awkward_kaitai
+
+
+def test_index_option():
+    reader = awkward_kaitai.Reader("test_artifacts/libindex_option.so")
+    awkward_array = reader.load("example_data/data/index_option.raw")
+
+    expected_array = np.array(
+        [(100, 16)],
+        dtype=[("bank_hdrA__Zall_bank_size", "<u4"), ("bank_hdrA__Zflags", "<u4")],
+    )
+
+    np.testing.assert_array_equal(
+        awkward_array.index_optionA__Zbank_header.to_numpy(), expected_array
+    )
+
+    expected_array_bank1 = {
+        "0": np.ma.masked_array(data=[], mask=[], dtype=np.uint8).reshape((1, 0)),
```

Comment:
@ManasviGoyal I find it suspicious that field "0" has a different shape than field "1" and "2", do you think this is expected in how you generated the data file?